repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoopInterpreter.scala

```diff
@@ -21,8 +21,22 @@ import scala.collection.mutable
 import scala.tools.nsc.Settings
 import scala.tools.nsc.interpreter._
 
-class SparkILoopInterpreter(settings: Settings, out: JPrintWriter) extends IMain(settings, out) {
-  self =>
+class SparkILoopInterpreter(settings: Settings, out: JPrintWriter, initializeSpark: () => Unit)
+    extends IMain(settings, out) { self =>
```

Comment:
nit: two spaces

Reply:
I thought for `extends`, it's four spaces?

Reply:
IIRC, four spaces is OK.

Reply:
Guys, parameters are listed in 4 spaces and other keywords after that are lined up with 2 spaces, which is written in https://github.com/databricks/scala-style-guide#spacing-and-indentation

In case of two lines, it's not explicitly written but wouldn't we better stick to the example as possible as we can?

Reply:
Well, there's a bunch of code in Spark where both 4-space and 2-space indentation are existed.  From my thought it is not so worthy to stick to this tiny point.

If you want to fix it, then creating a separate PR to fix all of them.

Reply:
We can fix them in place. There are many other style nits too. It's not worth swiping them making the backport harder.

I am fine with ignoring such nits and they don't block this PR but it's not something we should say the opposite side is okay.

Reply:
Really not worthy to block on this thing (if it is the only issue).